docs/faqs.md

```diff
@@ -3,15 +3,17 @@
 ##Cloning questions##
 *about satellite-clone-playbook.yml*
 
-####Where should I run Satellite Clone from?####
+####Can I run Satellite Clone from a different machine?####
 
-There are many options for where you can run Satellite Clone from (referenced as `Control Node` in the documentation). RHEL 6 and 7 machines are supported as that is what we test on, but it can run on any machine that has ansible installed and can communicate with your blank machine you are cloning to (referenced as `Destination Node` in the documentation). It can even be run on the blank Destination Node itself. Here are some typical scenarios:
+There are many options for where you can run Satellite Clone from, but we specify and test a simple "one machine" installation. RHEL 7 machines are supported as that is what we test on, but Satellite Clone can run on any machine that has ansible installed and can communicate with your blank target machine you are cloning to. Here are some scenarios:
 
-1. Use the blank destination node as the Control node. This is the "one machine" approach. You can clone the satellite-clone repo on this server, move the backups to the specified folder on the server, and then run the playbook from the repo.
+1. Use the blank target machine as the Ansible Control node. This is the "one machine" approach. You can clone the satellite-clone repo on this server, move the backups to the specified folder on the server, and then run the playbook from the repo. This is the scenario we detail in the docs.
 
-2. Use the Satellite you are cloning as the Control node. You can clone the satellite-clone repo on the existing Satellite server and use that as the Control Node. The Destination Node is a blank machine.
+2. Use the Satellite you are cloning as the Ansible Control node. You can clone the satellite-clone repo on the existing Satellite server and use that as the Control Node. The target machine is a blank target machine.
 
-3. Use a separate machine as the Control Node. This is a machine that is seperate from the blank machine you are cloning to and the Satellite you are cloning. The backups will have to be copied onto the server. The Destination Node is a blank machine.
+3. Use a separate machine as the Ansible Control Node. This is a machine that is seperate from the blank target machine you are cloning to and the Satellite you are cloning. The backups will have to be copied onto the blank target machine.
+
+If you are going to try any of the alternative scenarios, be sure to update the inventory file to point to the correct machine.
 
```

Comment:
Approach 2 and 3 here are the unsupported/untested use cases after this PR merges.  Why should we list something which we don't test or support?  Instead specify the one machine approach in detail and just mention that this playbook can run from any Ansible control node.  I assume, if a user wants to try other use cases, he should be familiar with using ansible.